R/serialization.R

```diff
@@ -0,0 +1,41 @@
+# This file will be autogenerated.
+
+#' Serialize a model to the given filename.
+#'
+#' @title Serialize.
+#' @param filename Input filename.
+#' @param model_in Input model.
+Serialize <- function(filename, model_in) {
```

Comment:
To avoid complications with _R_'s `serialize()`/`unserialize()`, let's prefix functions with:

`mlpack_serialize()`/`mlpack_unserialize()`,

Reply:
I think if push came to shove I would vote against that.  Capitalized alternates are fairly common around R -- `Ecdf()` (by Frank Harrell) has generalized `ecdf()` (in base R)) for decades.  Simpler is better (up to a point) and `Serialize` != `serialize`.